src/requirements.txt

```diff
@@ -0,0 +1,2 @@
+boto3==1.9.219
+elasticsearch7==7.13.0
```

Comment:
These are the requirements for the application. There is a separate file for requirements for testing.